packages/sdk/src/handlers/contract-kit.ts

```diff
@@ -12,14 +14,21 @@ import { ChainHandler } from './interface';
  */
 export class ContractKitTransactionHandler implements ChainHandler {
   private signedTransaction?: EncodedTransaction;
-  private readonly blockchainAddress;
-  private readonly dekAddress;
+  private readonly blockchainAddress: string;
+  private dekAddress;
 
   constructor(private readonly kit: ContractKit) {
     [this.blockchainAddress, this.dekAddress] = this.kit
```

Comment:
We should figure out how to get the typing correct so we can remove this dekAddress assignment. We'll probably need to visit how we can wait for the dekAddress. It might involve some api changes which at this point is fine.